go/libraries/events/event_flush_test.go

```diff
@@ -73,13 +95,62 @@ func TestEF(t *testing.T) {
 
 			assert.Equal(t, len(ces), test.numEvents)
 
-			err := fbp.WriteEvents(version, ces)
+			err := flushTester.Fbp.WriteEvents(testVersion, ces)
+			assert.Equal(t, err, nil)
+
+			err = flushTester.Flusher.FlushEvents(ctx)
+
+			assert.Equal(t, err, nil)
+			assert.Equal(t, len(flushTester.Client.CES), len(ces))
+		})
+	}
+}
+
+func TestEFLocal(t *testing.T) {
+	tests := []struct {
+		name      string
+		numEvents int
+	}{
+		{
+			name:      "Flush 0 events",
+			numEvents: 0,
+		},
+		{
+			name:      "Flush 100 events",
+			numEvents: 100,
+		},
+		{
+			name:      "Flush 1000 events",
+			numEvents: 1000,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			fs := filesys.LocalFS
+
+			path := filepath.Join(dPath, evtPath)
+
+			dDir := testLib.TestDir(path)
+
+			ctx := context.Background()
+
+			flushTester := createFlushTester(fs, "", dDir)
+
+			ces := make([]*eventsapi.ClientEvent, 0)
+
+			for i := 0; i < test.numEvents; i++ {
+				ce := &eventsapi.ClientEvent{}
+				ces = append(ces, ce)
+			}
+
+			err := flushTester.Fbp.WriteEvents(testVersion, ces)
 			assert.Equal(t, err, nil)
 
-			err = flush.FlushEvents()
+			err = flushTester.Flusher.FlushEvents(ctx)
 
 			assert.Equal(t, err, nil)
-			assert.Equal(t, len(client.CES), len(ces))
+			assert.Equal(t, len(flushTester.Client.CES), len(ces))
 		})
 	}
 }
```

Comment:
This is the new test for the local filesys. I will figure out a way to not duplicate test logic here as these tests are largely the same...